.github/workflows/ci.yml

```diff
@@ -15,13 +15,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: ['3.8']
-        toxenv: ["py38", "quality"]
-
+          python-version: ['3.12', '3.11']
+          toxenv: ["py312", "py311", "quality"]
```

Comment:
Style: Not a blocker but generally, having the older python version listed first looks nicer and cleaner.